Makefile

```diff
@@ -179,6 +179,12 @@ proto: docker-build-proto
 	| docker run -i pachyderm_proto \
 	| tar xf -
 
+# Use this to grab a binary for profiling purposes
+grab-binary: docker-clean-pachd docker-build-compile
```

Comment:
Small nitpick, could we call this `pachd-binary`. `make grab-binary` reads weirdly while `make pachd-binary` seems much nicer. In general I think make targets shouldn't include verbs since `make` is already the verb.